modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
+
+You can configure the Ingress Controller to enable mutual TLS (mTLS) authentication by setting a `spec.clientTLS` value. The `clientTLS` field specifies configuration for the Ingress Controller to verify client certificates. This configuration includes setting a `clientCA` value, which is a reference to a config map that contains the PEM-encoded CA certificate bundle that is used to verify a client's certificate, as well as an optional list of certificate subject filters.
+
+If the `clientCA` value specifies an X509v3 certificate revocation list (CRL) distribution point, the Ingress Controller downloads and enforces the CRL. Requests that do not provide valid certificates are rejected.
```

Comment:
Technically, it's the operator that downloads the CRL, not the ingress controller.  
```suggestion
If the `clientCA` value specifies an X509v3 certificate revocation list (CRL) distribution point, the Ingress Operator downloads the CRL and configures the Ingress Controller to respect it. Requests that do not provide valid certificates are rejected.
```